.github/workflows/e2e.yaml

```diff
@@ -84,13 +84,16 @@ jobs:
       - name: Build agent-enclave bundle
         run: |
           mkdir $PAYLOAD_ARTIFACTS
-          ./gen_bundle.sh agent-enclave-bundle:build
+          docker build ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc -f ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc/tools/packaging/build/agent-enclave-bundle/Dockerfile --build-arg SGX_MODE=${SGX_MODE} -t agent-instance:build
+          jq -a -f sgx-mode-config.filter config.json.template | tee ${PAYLOAD_ARTIFACTS}/config.json
```

Comment:
just an idea but this `jq` command could also run in its own job target:

```
- name: Install config.json
  run: | 
    jq -a -f sgx-mode-config.filter config.json.template | tee ${PAYLOAD_ARTIFACTS}/config.json
   working-directory: ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc/tools/packaging/build/agent-enclave-bundle
```

Reply:
> just an idea but this `jq` command could also run in its own job target:
> 
> ```
> - name: Install config.json
>   run: | 
>     jq -a -f sgx-mode-config.filter config.json.template | tee ${PAYLOAD_ARTIFACTS}/config.json
>    working-directory: ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc/tools/packaging/build/agent-enclave-bundle
> ```

Yeah, but this command has dependency on the "mkdir $PAYLOAD_ARTIFACTS" operation done in the "build agent_enclave bundle" job, and the command target is also related to prepare part of the bundle content. So I would prefer to keep them together to be easily understood.